### PR TITLE
Feat/CIV-231 where does the defendant live

### PR DESCRIPTION
**Pull request by dtortolaV1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-231

### Change description ###
Screen to add the type of the defendant's home

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json [hunk 1]
```diff
@@ -101,5 +101,26 @@
...
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "homeOptionsTitleLRspec",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "defendantHomeOptions",
+    "ShowSummaryChangeOption": "N",
```
Comment (sampankumar) [inline ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json:113]: If ShowSummaryChangeOption is N then this property wont show on the Check your Answers screen.
I think in the end we would need to show this on that screen so you can make this as Y

File: ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json [hunk 2]
```diff
@@ -101,5 +101,26 @@
...
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "respondent1DQHomeDetails",
+    "PageDisplayOrder": 13,
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "defendantHomeOptions",
+    "ShowSummaryChangeOption": "N"
```
Comment (sampankumar) [inline ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json:124]: If ShowSummaryChangeOption is N then this property wont show on the Check your Answers screen.
I think in the end we would need to show this on that screen so you can make this as Y

File: ccd-definition/CaseEventToComplexTypes/HomeDetailsLRspec.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "ID": "HomeDetailsLRspec",
```
Comment (bhssamray) [inline ccd-definition/CaseEventToComplexTypes/HomeDetailsLRspec.json:3]: If all sections of page are mandatory and you have marked DisplayContext = Mandatory in CaseEventtoFields, my understanding is that you do not need to make any change to this file. Please can you revert the changes to this file and check if it still works.

Comment (dtortolaV1) [reply ccd-definition/CaseEventToComplexTypes/HomeDetailsLRspec.json:3]: You're right, it works. I'll remove this file then

**Review by bhssamray — DISMISSED** [review]

File: ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json [hunk 4]
```diff
@@ -241,5 +241,26 @@
     "PageFieldDisplayOrder": 2,
     "DisplayContext": "COMPLEX",
     "ShowSummaryChangeOption": "Y"
+  },
```
Comment (kenneth-hmcts) [inline ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json:244]: Is it better to put these code after Page with PageID DefendantBankAccounts (PageDisplayOrder 11) in stead of putting it after Page with Page ID DefendantDebts (PageDisplayOrder 19) ? As it is PageDisplayOrder13.

**Review by kenneth-hmcts — APPROVED** [review]

**Review by bhssamray — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]
